build.gradle

```diff
@@ -161,6 +161,7 @@ task functional(type: Test) {
     include "uk/gov/hmcts/reform/sscs/functional/**"
     exclude "uk/gov/hmcts/reform/sscs/smoke/**"
     exclude("uk/gov/hmcts/reform/sscs/smoke/**", "uk/gov/hmcts/reform/sscs/functional/CohNotificationFunctionalTest")
+    exclude("uk/gov/hmcts/reform/sscs/smoke/**", "**/*CohNotificationFunctionalTest*")
```

Comment:
Do we need both this new line (164) and line 163? They seem very similar. Do they both work?